markets/spot-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -138,31 +142,6 @@ contract AsyncOrderModule is IAsyncOrderModule {
         }
     }
 
-    function settleOnChainOrder(
```

Comment:
this was unnecessary, you'd just call settle with the claim id and it will settle on chain if thats the strategy used